packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -141,7 +141,6 @@ spec:
             --wsaddr 0.0.0.0 \
             --wsorigins=* \
             --wsapi=eth,net,web3,debug \
-            --nodekey=/root/.celo/pkey \
```

Comment:
It looks like in this setup we would still have the private key at /root/.celo/pkey. Where is this file created?

Reply:
So I couldn't find a good way of never writing it to disk. But I made sure it gets deleted.